### DISCUSSION TITLE
deploy next app with yarn 1.22.19 instead of 1.22.17

### DISCUSSION THREAD

**Discussion by MiDonGo64** [vercel/community#2116] [Help] [answered]
### Summary

I build my project with yarn 1.22.19 and node 16.15.0 and can't find any information on how to properly set these variables in the build. Could be that vercel doesn't support past yarn 1.22.17 yet but would like to know how to set these production variables. I'm new to Next and Vercel. 

Thank you.

**Comment by amyegan** [comment, marked as answer]
Hi @MiDonGo64! We have experimental Corepack support to let you set a package manager version in your Project. Vercel automatically rolls out minor and patch updates as needed to address security and other issues, so only major Node versions can be selected.

You can find more info in the [Corepack](https://vercel.com/docs/concepts/deployments/configure-a-build#corepack) and [Node.js Versions](https://vercel.com/docs/concepts/functions/serverless-functions/runtimes/node-js#node.js-version) documentation
